environment.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+import cv2
+import gymnasium as gym
+import numpy as np
+from gymnasium.spaces import Box, Dict, OneOf
+
+import client
+
+
+class TankwarsEnv(gym.Env):
+
+    def __init__(self, client):
+        self.client = client
+        self.observation_space = Box(0, 255, shape=(200, 200, 3), dtype=np.uint8)
+        self.action_space = Dict({
+            "right_engine": Box(-1, 1, shape=(1,), dtype=np.float32),
+            "left_engine": Box(-1, 1, shape=(1,), dtype=np.float32),
+            "fire": OneOf(True, False),
```

Comment:
I don't think this is the proper usage of OneOf, we should use a Box(dtype=bool) instead